### PR TITLE
Updated version of rss component and fixed content picking plus tests

### PR DESCRIPTION
**Pull request by rodrigopavezi**
@donnapep @stulees Please review. 

Presentation here: 
http://rva.risevision.com/#/PRESENTATION_MANAGE/id=be0200ca-e57c-475f-a525-1c1dc0de857d?cid=30007b45-3df0-4c7b-9f7f-7d8ce6443013

Cheers

### CODE REVIEW

**Comment by donnapep** [comment]
:+1:

File: test/integration/rss-none-transition.html [hunk 1]
```diff
@@ -80,29 +79,29 @@
...
 
         assert.equal(document.querySelector(".headline a").innerHTML,
-          "RSS 2.0 – Entry 2 title", "second still item displaying");
+          "RSS 2.0 - Entry 2 title", "second still item displaying");
 
         clock.tick(5000);
 
         assert.equal(document.querySelector(".headline a").innerHTML,
-          "RSS 2.0 – Entry 3 title", "Third item displaying");
+          "RSS 2.0 - Entry 3 title", "Third item displaying");
```
Comment (stulees) [inline test/integration/rss-none-transition.html]: @rodrigopavezi Noticing a lot of differences regarding "-" dash characters, any reason?

Comment (rodrigopavezi) [inline test/integration/rss-none-transition.html]: @stulees @donnapep changed the tests when she changed the dada on this PR https://github.com/Rise-Vision/widget-rss/pull/34/files. Now I have changed the data again. And I am taking the data directly from the feed parser output. cheers

**Comment by stulees** [comment]
@rodrigopavezi :+1:

**Pull request merged**